package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "flashpointbrowser",
+  "version": "0.0.0",
```

Comment:
Let's say 0.0.1?

Reply:
Sure, though I was just going to let you update that. Your branch before my changes would be 0.0.1, and after would be 0.0.2?

Reply:
I don't really care as we've not been really tracking version so far. 0.0.1 works for me.

Reply:
Ok, I'll just do that then.